Mon Jan 16 17:17:12 GMT 2017 Thread[main,5,main] Cleanup action starting
java.sql.SQLException: Database 'TillEmbedded' not found.
	at org.apache.derby.impl.jdbc.SQLExceptionFactory40.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.handleDBNotFound(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection40.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver40.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.Driver20.connect(Unknown Source)
	at org.apache.derby.jdbc.AutoloadedDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at io.github.davidg95.JTill.jtill.DBConnect.connect(DBConnect.java:94)
	at io.github.davidg95.JTill.jtillserver.GUI.databaseLogin(GUI.java:90)
	at io.github.davidg95.JTill.jtillserver.TillServer.start(TillServer.java:77)
	at io.github.davidg95.JTill.jtillserver.TillServer.main(TillServer.java:55)
Caused by: java.sql.SQLException: Database 'TillEmbedded' not found.
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory40.wrapArgsForTransportAcrossDRDA(Unknown Source)
	... 18 more
============= begin nested exception, level (1) ===========
java.sql.SQLException: Database 'TillEmbedded' not found.
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory40.wrapArgsForTransportAcrossDRDA(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory40.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.handleDBNotFound(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection40.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver40.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.Driver20.connect(Unknown Source)
	at org.apache.derby.jdbc.AutoloadedDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at io.github.davidg95.JTill.jtill.DBConnect.connect(DBConnect.java:94)
	at io.github.davidg95.JTill.jtillserver.GUI.databaseLogin(GUI.java:90)
	at io.github.davidg95.JTill.jtillserver.TillServer.start(TillServer.java:77)
	at io.github.davidg95.JTill.jtillserver.TillServer.main(TillServer.java:55)
============= end nested exception, level (1) ===========
Cleanup action completed
----------------------------------------------------------------
Mon Jan 16 17:17:19 GMT 2017:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.10.1.3 - (1557168): instance a816c00e-0159-a848-5f0d-00000664d320 
on database directory C:\Users\David\Honors Project\Till\JTillServer\TillEmbedded with class loader sun.misc.Launcher$AppClassLoader@58644d46 
Loaded from file:/C:/Users/David/Honors%20Project/Till/JTillServer/derby/derby.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_31-b13
user.dir=C:\Users\David\Honors Project\Till\JTillServer
os.name=Windows 8.1
os.arch=amd64
os.version=6.3
derby.system.home=null
Database Class Loader started - derby.database.classpath=''
